.github/workflows/ci.yaml

```diff
@@ -40,4 +44,39 @@ jobs:
 
     - name: push packages
       if: github.event_name == 'push' && github.ref == 'refs/heads/master'
-      run: dotnet nuget push **/*.nupkg -s https://f.feedz.io/davidfowl/featherhttp/nuget/index.json -ss https://f.feedz.io/davidfowl/featherhttp/symbols --skip-duplicate
\ No newline at end of file
+      run: dotnet nuget push **/*.nupkg -s https://f.feedz.io/davidfowl/featherhttp/nuget/index.json -ss https://f.feedz.io/davidfowl/featherhttp/symbols --skip-duplicate
+    
+    - uses: actions/checkout@master
+      if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+      with:
+        ref: ${{ github.ref }} 
+
+    - name: nuget version update
+      if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+      run: | 
+        powershell ./update-nuget-references.ps1
+        git status        
+
+    - name: Commit files
```

Comment:
What's this for?

Reply:
This existed because I misunderstood the requirements. This has now been changed

Reply:
So specifically - what I thought you wanted was to get the version info and update the csproj files in mater with that version info. 
But after looking at other implementations at other projects and reading and thinking more about what you may want - I thought you probably just want the uploaded nuget file csproj to have the correct version and there is no reason to do a commit from the ci build. 
Tbh, I had misunderstood the requirement for this issue and that's why the PR has changed so much. I think it's pretty close in it's current state